Cargo.toml

```
@@ -10,24 +10,24 @@ license = "MIT OR Apache-2.0"
 {{ mcu }}-hal = "0.7.0"
 {% else -%}
 {{ mcu }}-hal = "0.4.0"
```

```suggestion
{{ mcu }}-hal = "0.8.0"
{% else -%}
{{ mcu }}-hal = "0.5.0"
```

We've had new releases of the HALs since this PR was opened